test/zzz-rsyslog.sh

```diff
@@ -274,7 +275,7 @@ if [ $logsize -gt 0 ]; then
     oc exec $rpod -c rsyslog -- ls -l /var/log/rsyslog/ | artifact_out
     oc exec $rpod -c rsyslog -- cat /var/log/rsyslog/rsyslog.log > $ARTIFACT_DIR/rsyslog-before-rotation.log
     # this will restart the rsyslog pod
-    oc set env daemonset/rsyslog LOGGING_FILE_SIZE=$maxsize LOGGING_FILE_AGE=$maxcount
+    oc set env $rsyslog_ds LOGGING_FILE_SIZE=$maxsize LOGGING_FILE_AGE=$maxcount
```

Comment:
```
/go/src/github.com/openshift/origin-aggregated-logging/test/zzz-rsyslog.sh: line 278: rsyslog_ds: unbound variable
```

Reply:
`$rsyslog_ds` does not have the correct value:
the zzz-rsyslog test failed:
```
F0709 18:21:33.913427  108255 helpers.go:119] Error from server (NotFound): daemonsets.extensions "logging-rsyslog" not found
```
should be `rsyslog` not `logging-rsyslog`

Reply:
I can revert it.  But why this does not work?  I 

+get_rsyslog_ds_name() {
+    if oc -n ${LOGGING_NS} get daemonset rsyslog -o name > /dev/null 2>&1 ; then
+        echo daemonset/rsyslog
+    else
+        echo daemonset/logging-rsyslog
+    fi
+}
+
+rsyslog_ds=${rsyslog_ds:-$(get_rsyslog_ds_name)}

Reply:
because `daemonset rsyslog` does not exist when `get_rsyslog_ds_name` is called?

Reply:
I see.  That's possible...  Thanks.  I'm going to change the code as you suggested.

> I think you should just hardcode this to be ds/rsyslog